_src/css/_header.scss

```diff
@@ -6,9 +6,12 @@
   float: left;
   background-image: url('../assets/taina.png');
   background-repeat: no-repeat;
-  background-position: bottom center;
   background-size: cover;
```

Comment:
@felipprodrigues `background-size: cover` should be moved to that media query as well, no?